components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -397,3 +409,79 @@ function OrgDefaultWorkspaceImageModal(props: OrgDefaultWorkspaceImageModalProps
         </Modal>
     );
 }
+
+interface WorkspaceSizeOptionsProps {
+    settings: OrganizationSettings | undefined;
+}
+const WorkspaceSizeOptions = (props: WorkspaceSizeOptionsProps) => {
+    const [errorMsg, setErrorMsg] = useState("");
+    const [selectedValue, setSelectedValue] = useState(props.settings?.maximumWorkspaceClass ?? "");
+    const updateTeamSettings = useUpdateOrgSettingsMutation();
+    const { data: classes, isError, isLoading } = useWorkspaceClasses();
+
+    const { toast } = useToast();
+
+    const handleUpdateTeamSettings = useCallback(
+        async (cls: string) => {
+            try {
+                await updateTeamSettings.mutateAsync(
+                    {
+                        ...props.settings,
+                        maximumWorkspaceClass: cls,
+                    },
+                    {
+                        onSuccess: () => {
+                            toast({ message: "Organization workspace size updated" });
```

Comment:
Current version: 

![image](https://github.com/gitpod-io/gitpod/assets/55068936/ab537d68-a488-4695-877f-f5ef55c16647)


```suggestion
                            toast({ message: "Workspace size limit updated for your organization." });
```